src/qml/pages/node/NodeSettings.qml

```diff
@@ -13,73 +13,72 @@ Item {
     id: nodeSettings
     property alias navMiddleDetail: nodeSettingsView.navMiddleDetail
     property alias navRightDetail: nodeSettingsView.navRightDetail
+
     StackView {
         id: nodeSettingsView
         property alias navMiddleDetail: node_settings.navMiddleDetail
         property alias navRightDetail: node_settings.navRightDetail
+        anchors.fill: parent
+
         initialItem: Page {
             id: node_settings
             property alias navMiddleDetail: navbar.middleDetail
             property alias navRightDetail: navbar.rightDetail
             background: null
+            implicitWidth: 490
```

Comment:
This is required due to an oddity with the StackView initial state. It will be ignored with the anchor properties.